Watchman.Discord/Areas/Protection/Services/UnmutingService.cs

```diff
@@ -84,6 +84,8 @@ public async Task UnmuteNow(Contexts contexts, UserContext userToUnmute)
             if (eventToUnmute == null)
             {
                 Log.Information("{userName} is not muted", userToUnmute.Name);
+                var messagesService = this._messagesServiceFactory.Create(contexts);
+                messagesService.SendResponse(x => x.UserWasntMuted(userToUnmute));
```

Comment:
ja bym tu jednak dał await, mimo, że nie stanowi tu to dużego problemu.

Reply:
faktycznie, nie zauważyłem